boot/freeldr/freeldr/arch/uefi/uefimem.c

```diff
@@ -306,7 +326,6 @@ UefiExitBootServices(VOID)
 VOID
 UefiPrepareForReactOS(VOID)
 {
-    UefiExitBootServices();
-    ExitStack = MmAllocateMemoryWithType(EXIT_STACK_SIZE, LoaderOsloaderStack);
-    EndofExitStack = (PVOID)((ULONG_PTR)ExitStack + EXIT_STACK_SIZE);
+    _exituefi();
+    _changegdt();
```

Comment:
It looks like these two helper functions could be combined into one? Or do you keep them separated because you think you may need to use them sometimes separately, in the future?